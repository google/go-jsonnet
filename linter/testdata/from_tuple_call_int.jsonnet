local arr = [function() 42, 42];
[
    arr[0](),
    arr[1](),
]
