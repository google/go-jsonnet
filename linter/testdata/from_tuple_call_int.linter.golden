testdata/from_tuple_call_int:4:5-13 Called value must be a function, but it is assumed to be a number

    arr[1](),


